Dockerfile

```diff
@@ -13,14 +12,22 @@
 
 ENV PYTHONUNBUFFERED=1
 
+# Install system dependencies for mysqlclient and netcat
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    default-libmysqlclient-dev \
+    pkg-config \
+    netcat-openbsd && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
 COPY --from=builder /app .
```

Comment:
The `COPY --from=builder /app .` command assumes that the build stage has produced some artifacts that need to be copied to the run stage. Ensure that the build stage actually generates these artifacts, or adjust the build process accordingly.